content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/moves_history.rst

```diff
@@ -0,0 +1,100 @@
+- :guilabel:`Unit`: unit of measure of the products moved.
+- :guilabel:`Status`: indicates the move status, which can be :guilabel:`Done`,
+  :guilabel:`Available` (ready for action), or :guilabel:`Partially Available` (insufficient
+  quantities to complete the operation).
+
+Search options
+--------------
+
+Use the following search options to customize the :guilabel:`Moves History` report to display
+relevant information
+
+.. tabs::
+
+   .. tab:: Filters
+
+      The :guilabel:`Filters` section allows users to search among pre-made and custom filters to
+      find specific stock records.
+
+      - :guilabel:`To Do`: show stock move records that are in progress. This includes lines with a
+        :guilabel:`Status` column value of :guilabel:`Available` or :guilabel:`Partially Available`.
+      - :guilabel:`Done`: completed stock moves, with a :guilabel:`Status` of :guilabel:`Done`.
+      - :guilabel:`Incoming`: displays move records from vendor locations.
+      - :guilabel:`Outgoing`: displays move records to customer locations, including customer
+        returns.
+      - :guilabel:`Internal`: displays move records from one internal location to another.
+      - :guilabel:`Manufacturing`: shows records where products were produced from the virtual,
+        production location.
+
+        .. seealso::
+           :doc:`../inventory_management/use_locations`
+
+      - :guilabel:`Date`: select this drop-down menu to access various date filter options and view
+        stock moves from a specific month, quarter, or year.
+      - :guilabel:`Last 30 Days`: show records that occurred in the last thirty days.
+      - :guilabel:`Last 3 Months`: show records from the last three months.
+
+   .. tab:: Group By
+
+      The :guilabel:`Group By` section allows users to add pre-made and custom groupings to the
+      search.
+
+      - :guilabel:`Product`: group records by product.
+      - :guilabel:`Status`: group records by the three status types: :guilabel:`Done`,
+        :guilabel:`Available`, and :guilabel:`Partially Available`.
+      - :guilabel:`Date`: group records by :guilabel:`Year`, :guilabel:`Quarter`, :guilabel:`Month`,
+        :guilabel:`Week`, or :guilabel:`Day`.
+      - :guilabel:`Transfers`: group records by operation number, e.g. `WH/OUT/00012`,
+        `WH/MO/00211`.
+      - :guilabel:`Location`: group records by source location (the :guilabel:`From` column in this
+        report)
+      - :guilabel:`Category`: group records by product category. To configure these, go to
+        :menuselection:`Inventory app --> Configuration --> Products: Product Categories`.
+      - :guilabel:`Batch Transfer`: group records by :doc:`batch
+        <../advanced_operations_warehouse/batch_transfers>`
+
+   .. tab:: Favorites
+
+      To save the current applied filters and groupbys, so the same information can be easily
+      accessed after closing this page, click :guilabel:`Save current search`.
```

Comment:
Nitpick– I think you can just say "groups" instead of "groupbys"
```suggestion
      To save the current applied filters and groups, so the same information can be easily accessed
      after closing this page, click :guilabel:`Save current search`.
```